source/_integrations/matter.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Matter
+description: Instructions on how to integrate Matter with Home Assistant.
+featured: true
+ha_category:
+  - Light
+ha_release: '2022.11'
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@MartinHjelmare'
+ha_domain: matter
+ha_platforms:
+  - light
+ha_integration_type: integration
+---
+
+This integration allows you to control a Matter network.
```

Comment:
I'd rephrase this a bit so it will be about controlling Matter devices and not a Matter network.

Reply:
```suggestion
This integration allows you to communicate with and control Matter devices.
```